spec/rspec/expectations/configuration_spec.rb

```diff
@@ -47,7 +45,15 @@ class << rspec_dup; undef configuration; end
       end
 
       context 'on an interpreter that does not provide BasicObject', :uses_should, :unless => defined?(::BasicObject) do
-        before { RSpec::Expectations::Syntax.disable_should(Delegator) }
+        include RSpec::Support::InSubProcess
+
+        around do |example|
+          RSpec::Expectations::Syntax.disable_should(Delegator)
```

Comment:
Given `require 'delegate'` now comes after this line, and `Delegator` is defined in the `delegate` stdlib...won't this fail?